.github/actions/templating/action.yml

```diff
@@ -0,0 +1,11 @@
+name: templating
```

Comment:
Why is this called "templating"?

Reply:
because files are generated from template, term used for ex. by official https://github.com/docker-library/php/tree/master/.github/workflows php images repo

Reply:
is this name ok with you, or would you prefer something like `verify-templating` to stress this is for verification/read only?